Fix:
error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strncmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
https://github.com/w8tvi/dircproxy/issues/80
--- getopt/getopt.c.orig	2009-01-15 14:27:28.000000000 -0600
+++ getopt/getopt.c	2022-01-14 15:48:54.000000000 -0600
@@ -52,6 +52,7 @@
 #endif
 
 #include <stdio.h>
+#include <string.h>
 
 /* Comment out all this code if we are using the GNU C Library, and are not
    actually compiling the library itself.  This code is part of the GNU C
